### DISCUSSION TITLE
plans for 1.0.0 release?

### DISCUSSION THREAD

**Discussion by scarf005** [c4spar/cliffy#753] [Q&A] [answered]
while #680 is resolved, cliffy still cannot be easily installed due to not having stable release (see: https://github.com/jsr-io/jsr/issues/745). it'd be great to know when would it be released as stable 1.0.0.

**Comment by c4spar** [comment, marked as answer]
I need to update the documentation/website to reflect the changes for jsr. I will do the 1.0 release after that.
